### PR TITLE
Change predict functions to `predict()` and `predictTopK()` in pose and image libraries

### PR DESCRIPTION
**Pull request by irealva**
Also added a tiny bit of cleanup to pose by removing an unused `flipped` boolean.

### CODE REVIEW

File: pose/README.md [hunk 1]
```diff
@@ -201,15 +201,13 @@ This method exists on the model that is loaded from `tmPose.load`.
 ```ts
 model.predict(
     poseOutput: Float32Array,
```
Comment (hapticdata) [inline pose/README.md]: why was `flipped` taken off of `predict` ? This makes the method signatures between pose and image different which is confusing.

Comment (irealva) [reply pose/README.md]: Just logging here we talked about it and are going to go with 1) function name changes and 2) moving the flag as last variable.

**Comment by irealva** [comment]
Have now added a test for both predict functions in image and pose libraries

File: pose/README.md (hunk 1 above)
Comment (irealva) [reply pose/README.md]: Now the functions are `predict` and `predictTopK`

**Comment by irealva** [comment]
One comment about the order of variables.

In image:
`predict(image, flipped)`
`predictTopK(image, flipped, maxPredictions)`

in pose:
`predict(posenetOutput)`
`predictTopK(posenetOutput, maxPredictions)`

The problem with putting the flipped boolean at the end of the `predictTopK` in image lib is that then the two predict functions don't match within the same library which is even more confusing.

**Pull request merged**